content/kips/kip-19.md

```diff
@@ -0,0 +1,83 @@
+---
+kip: 19
+title: Advanced Orders
+status: Draft
+author: Jeremy Chiaramonte
+created: 2022-06-30
+---
+
+## Summary
+
+Advanced orders introduce to new trading strategies for futures.
+
+## Abstract
+
+Leveraging Gelato's keeper system, Kwenta will enable the automation of orders for futures. The first two products to come of this will be Limit Orders & Stop Market Orders. 
+
+## Motivation
+
+Market orders are the only available order type for Kwenta futures at the moment. There are trading benefits by allowing orders to execute in the future based on defined conditions. 
```

Comment:
Next price is _technically_ a market order but this feels like it could be misinterpreted by a reader.